unity-client/Assets/UnityGLTF/Scripts/Loader/WebRequestLoader.cs

```diff
@@ -50,10 +48,7 @@ public IEnumerator LoadStream(string filePath)
             yield return CreateHTTPRequest(_rootURI, filePath);
         }
 
-        public void LoadStreamSync(string jsonFilePath)
-        {
-            throw new NotImplementedException();
-        }
+        public void LoadStreamSync(string jsonFilePath) { throw new NotImplementedException(); }
 
```

Comment:
I know that you didn't make this change but do you know if this Is suppose to throw not implemented?

Reply:
The truth is that I don't know.